packages/apollos-data-prayer/src/prayer-requests/__tests__/resolver.test.js

```diff
@@ -116,13 +122,15 @@ describe('PrayerRequest', () => {
   it('gets all public prayer requests by campus', async () => {
     const query = `
       query {
-        getPublicPrayerRequestsByCampus(campusId: 16) {
+        getPublicPrayerRequestsByCampus(campus: Campus:772fcb6087247ebad630814e2ce0cd16) {
```

Comment:
So I noticed most of the graphql queries y'all have added follow this format of "getX". This is somewhat of an anti-pattern in graphql. Think of a query as a node in the graph - they represent an object, not a function. A more graphql-esque name for this would be something like `prayerRequests` or `publicPrayerRequests`

The "functional" naming comes in when you create a query with the `query` syntax:

```
query getPublicPrayerRequestsByCampus {
   prayerRequests(campus: "blah") {
      ...attr
   }
}
```

Since most of the queries you've added follow this `getX` naming structure, I wouldn't change it in this PR, but when y'all PR against core, we will be evaluating the schema structure, so food for thought.

Reply:
yes love this! I was wondering how to make the client code simpler and I had this exact question "why doesn't my code line up semantically with Apollo's examples" this explains it.

@IsaacHardy I think we make this change, not before Thursday, but at some point to line up with Apollo and Apollos :)